d2/src/main/java/com/linkedin/d2/balancer/dualread/DualReadLoadBalancerMonitor.java

```diff
@@ -136,7 +129,7 @@ else if (!isDataEqual && isVersionEqual)
     { // data is not the same but version is the same, a mismatch!
       incrementPropertiesErrorCount();
       incrementEntryOutOfSyncCount(); // increment the out-of-sync count for the entry received later
-      warnByPropType(isUriProp,
+      logByPropType(isUriProp,
           String.format("Received mismatched %s for %s. %s", propertyClassName, propertyName, entriesLogMsg));
```

Comment:
```suggestion
String msg = String.format("Received mismatched %s for %s. %s", propertyClassName, propertyName, entriesLogMsg);
if (isUriProp)
{
    LOG.debug(msg);
}
else
{
    warnByPropType(isUriProp, msg);    
}
```

Reply:
in the isUriProp case, this should still probably be rateLimitedLogger right?

Reply:
also, with this version, this is actually equivalent to my proposed change. The isUriProp path in warnByLogType will be dead code since it's check prior to every invocation and DEBUG'd

Reply:
resolved offline, we will tune the rate limit for now instead of changing log level and explore better handling of long log lines

Reply:
`String msg = String.format("Received mismatch %s for %s. %s", propertyClassName, propertyName, entriesLogMsg);

if (isUriProp) {
    LOG.debug(msg);
} else {
    warnByPropType(isUriProp, msg);
}
`